src/NetworkViewUi.lua

```diff
@@ -136,6 +136,21 @@ local tab_idx_to_view_type = {
   "shortage",
 }
 
+local function item_tooltip(name, count)
+  return { "", game.item_prototypes[name].localised_name, ": ", count }
```

Comment:
What do you think about adding instructions for accessing items to the tooltip?

Reply:
"left click with an empty cursor to remove items"?

Reply:
I like the way that Factory Planner labels each key command on a separate line. I'd prefer to list every available key combination rather than try to keep the description short.

![image](https://github.com/year6b7a/item-network-factorio-mod/assets/135451123/69d4e6f0-6b1a-46e0-8429-a7817a2bc511)

Reply:
Feel free to update the tooltip text.  I'm likely not going to have time to play with this again until the weekend.